code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -15,6 +15,7 @@
 	obj_integrity = 150
 	max_integrity = 150
 	var/folder_path = /obj/item/weapon/folder //this is the path of the folder that gets spawned in New()
+	force_string = "professional"

```

Comment:
briefcase launchpad code is confusing me. It's a machine, so it can't be picked up so whats the point of tooltips?